pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -71,8 +71,9 @@ var defaultGatheringRules = []GatheringRule{
 		},
 		GatheringFunctions: GatheringFunctions{
 			GatherLogsOfNamespace: GatherLogsOfNamespaceParams{
-				Namespace: "openshift-cluster-samples-operator",
-				TailLines: 100,
+				Namespace:     "openshift-cluster-samples-operator",
+				TailLines:     100,
+				MaxContainers: 10,
```

Comment:
It was really hard to figure out the right number here. So, I must say that I'm not pretty sure about it. I tried to consider the size of the log per namespace, but even so, hard to come with a number.

I believe that 10 is a reasonable number, but, maybe we need to tune it later.

Reply:
Wasn't the idea to define this number within the condition?

Reply:
Well.... https://github.com/openshift/insights-operator/pull/557#discussion_r758418597